### ISSUE TITLE
Cannot disable rule from command line

### ISSUE THREAD

**Issue by Jorenar** [nvuillam/npm-groovy-lint#505]
I wanted to disable some rule via command line like so:
```sh
npm-groovy-lint --rulesetsoverridetype appendConfig --rulesets 'SomeRule{"enabled":false}' path/to/file.gradle
```

Moreover, when I tried with overriding config:
```sh
npm-groovy-lint --rulesets 'someRule{"enabled":false}' path/to/file.gradle
```
then not only the `SomeRule` was not disabled, but the only one visible.

**Comment by nvuillam** [comment]
@Jorenar once you use -ruleset argument, it is sent "as is" to CodeNarc, and .groovylintrc.json is ignored

The format is the following: https://codenarc.org/codenarc-creating-ruleset.html#creating-a-json-ruleset-file

If it's wrongly taken in account, the bug will have to be reported on CodeNarc side :/

But I strongly suggest to use .groovylintrc.json instead as it is handled directly by npm-groovy-lint :)

**Comment by Jorenar** [comment]
The config file is ignored, really? As a test, I've added to `.groovylintrc.json`:
```
        "ParameterName": {
            "regex": "[a-z]"
        },
```
then launched the linter with changing its severity:
```
npm-groovy-lint --rulesetsoverridetype appendConfig --rulesets 'ParameterName{"severity":"info"}' /path/to/file.gradle
```
and got what I expected:
```
xxx   info     The parameter named FOO in method <closure> of class None does not match [a-z]  ParameterName
```

And since presence of rule name in `--ruleset` triggers its activation, that made me suspect the problem occurs when merging the parameters.

**Comment by Jorenar** [comment]
Fix could be:
```diff
diff --git a/lib/codenarc-factory.js b/lib/codenarc-factory.js
index cc0f4fa..6e5648e 100644
--- a/lib/codenarc-factory.js
+++ b/lib/codenarc-factory.js
@@ -449,8 +449,8 @@ async function buildRuleSets(options) {
                       ? ruleOptions
                       : ruleFromConfig;
             const ruleDef = buildCodeNarcRule(ruleName, mergedRuleConfig);
-            // If rule has been sent as argument, enable it by default
-            if (ruleDef.enabled === false) {
+            // If rule has been sent as argument, enable it by default if not explicitly disabled
+            if (ruleOptions.enabled !== false && ruleDef.enabled === false) {
                 ruleDef.enabled = true;
             }
             return ruleDef;
```
